### PR TITLE
Refactored styling and markup throughout the app (#1xx0ebt)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: src/theme/variables.css [hunk 1]
```diff
@@ -371,6 +371,7 @@ hr {
 
   .sort > ion-item {
     width: 308px;
```
Comment (dt2patel) [inline src/theme/variables.css]: should this be max-width?

File: src/views/FindOrder.vue [hunk 2]
```diff
@@ -98,14 +98,14 @@
           </ion-card>
         </aside>
 
-        <main class="main">
+        <main class="main" @click="() => router.push('/order')">
```
Comment (dt2patel) [inline src/views/FindOrder.vue]: This the container for the list of orders. Users will click on orders not the list

File: src/views/FindOrder.vue [hunk 3]
```diff
@@ -98,14 +98,14 @@
...
           <section class="sort"></section>
 
-           <hr />
+          <hr />
 
           <section class="section-header">
```
Comment (dt2patel) [inline src/views/FindOrder.vue]: add code comment that this section is an order item to help developers.

File: src/views/FindProductInventory.vue [hunk 4]
```diff
@@ -119,7 +119,7 @@
            </ion-list>
         </aside>
 
-        <main class="main">
+        <main class="main" @click="() => router.push('/product-inventory')">
```
Comment (dt2patel) [inline src/views/FindProductInventory.vue]: same issue as order

File: src/views/FindProductInventory.vue [hunk 5]
```diff
@@ -174,9 +174,12 @@
 
               <div class="desktop-only">
                 <ion-list>
-                  <ion-list-header>Variants</ion-list-header>
-                  <div class="variant-detail">
-                    <div class="variant-info">
+                  <ion-list-header>
+                    Variants
+                    <hr align="right" width="90%" />
```
Comment (dt2patel) [inline src/views/FindProductInventory.vue]: I think this should be a css selector right? why inline?

File: src/views/FindProductInventory.vue [hunk 6]
```diff
@@ -193,14 +196,15 @@
                       </ion-chip>
                     </div>
 
-                    <div class="variant-metadata">
+                    <div>
                       <ion-item lines="none" detail>
                         <ion-note slot="end" class="metatags">3 variants</ion-note>
```
Comment (dt2patel) [inline src/views/FindProductInventory.vue]: what are we achieving with metatags class?

File: src/views/FindProductInventory.vue [hunk 7]
```diff
@@ -349,5 +336,14 @@ export default {
   .section-header {
   grid-template-columns: 1fr max-content 1fr;
```
Comment (dt2patel) [inline src/views/FindProductInventory.vue]: Indentation

File: src/views/FindPurchaseOrder.vue [hunk 8]
```diff
@@ -55,7 +55,7 @@
           </ion-list>
         </aside>
 
-        <main class="main">
+        <main class="main" @click="() => router.push('/purchase-order')">
```
Comment (dt2patel) [inline src/views/FindPurchaseOrder.vue]: same as FindOrder main element

File: src/views/FindPurchaseOrder.vue [hunk 9]
```diff
@@ -253,27 +258,17 @@ export default {
...
+@media (min-width: 991px) {
+  .list-item {
+    --columns-desktop: 3;
+    grid-template-columns: 1fr max-content 1fr;
+  }
 
-.variant-metadata {
-  grid-area: vmetadata;
-}
+  .variant-tags {
```
Comment (dt2patel) [inline src/views/FindPurchaseOrder.vue]: why do we need this class?

File: src/views/Order.vue [hunk 10]
```diff
@@ -275,70 +294,45 @@ export default {
...
+    grid: "image product"
+          / 240px 1fr;
   }
-  .desktop-only {
-    display: unset;
+
+  .product-image {
     grid-area: image;
     height: 362px;
+    border: 1px solid black;
```
Comment (dt2patel) [inline src/views/Order.vue]: instead of black we should use ionic colors

File: src/views/Order.vue [hunk 11]
```diff
@@ -275,70 +294,45 @@ export default {
...
+    border-radius: 10px;
```
Comment (dt2patel) [inline src/views/Order.vue:324]: we should come up with variables for border radius too.

Comment (dt2patel) [reply src/views/Order.vue:324]: doesn't have to be in this PR

File: src/views/ProductInventory.vue [hunk 12]
```diff
@@ -741,7 +741,7 @@ export default defineComponent({
 }
 
 .product-info,
-.variant-info {
+.variant-features {
```
Comment (dt2patel) [inline src/views/ProductInventory.vue]: info is better class name here

File: src/theme/variables.css [hunk 13]
```diff
@@ -396,4 +397,28 @@ hr {
...
+          "cards    timeline" 1fr
+          / 1fr 500px;
+    justify-content: space-between;  
+  }
+
+  .id {
+    grid-area: id;
+  }
+  
+  .cards {
```
Comment (dt2patel) [inline src/theme/variables.css]: cards is not a good class name, this is "info"

File: src/theme/variables.css [hunk 14]
```diff
@@ -396,4 +397,28 @@ hr {
...
+    display: grid;
+    grid-template-columns: repeat(2, minmax(314px, max-content));
```
Comment (dt2patel) [inline src/theme/variables.css]: we want to use auto repeat here, there can me more than just 2 columns.

**Review by dt2patel — CHANGES_REQUESTED** [review]
Please review changes

File: src/theme/variables.css (hunk 1 above)
Comment (azkyakhan) [reply src/theme/variables.css]: Yes sir , Done

File: src/views/FindOrder.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/FindOrder.vue]: Done

File: src/views/FindOrder.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/FindOrder.vue]: Ok sir

File: src/views/FindProductInventory.vue (hunk 4 above)
Comment (azkyakhan) [reply src/views/FindProductInventory.vue]: Done sir

File: src/views/FindProductInventory.vue (hunk 5 above)
Comment (azkyakhan) [reply src/views/FindProductInventory.vue]: Used selector here.

File: src/views/Order.vue (hunk 10 above)
Comment (azkyakhan) [reply src/views/Order.vue]: Ok sir

File: src/views/Order.vue (hunk 11 above)
Comment (azkyakhan) [reply src/views/Order.vue:324]: Ok

File: src/views/ProductInventory.vue (hunk 12 above)
Comment (azkyakhan) [reply src/views/ProductInventory.vue]: Reverted it back to `variant-info`

File: src/theme/variables.css (hunk 13 above)
Comment (azkyakhan) [reply src/theme/variables.css]: Ok sir

File: src/theme/variables.css (hunk 14 above)
Comment (azkyakhan) [reply src/theme/variables.css]: Ok

**Review by dt2patel — APPROVED** [review]
Looks good

**Pull request merged**